packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -220,6 +222,14 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu>, IWith
   /** Method to call when trying to render a submenu. */
   onRenderSubMenu?: IRenderFunction<IContextualMenuProps>;
 
+  /**
+   * Method to override the render of the individual menu items
+   * @default ContextualMenuItem
+   */
+  contextualMenuItemAs?:
```

Comment:
Is this a place we can use IComponentAs @dzearing ?

Reply:
Oh, IComponentAs isn't in 5.x yet. Should we pull it over? Or is this compatible so that we can update it in 6.0?